pom.xml

```diff
@@ -78,6 +78,9 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+                <!-- <configuration>
```

Comment:
Remove the commented out config please

Reply:
Resolved the issue and removed unused imports